src/main/java/pl/touk/parking/repository/converter/PaymentPolicyConverter.java

```diff
@@ -0,0 +1,33 @@
+package pl.touk.parking.repository.converter;
+
+import pl.touk.parking.PaymentPolicy;
+import pl.touk.parking.payment.DisabledDriverPaymentPolicy;
+import pl.touk.parking.payment.RegularDriverPaymentPolicy;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+@Converter(autoApply = true)
+public class PaymentPolicyConverter implements AttributeConverter<PaymentPolicy, String> {
+
+    private static final String REGULAR = "REGULAR";
+    private static final String DISABLED = "DISABLED";
+
+    @Override
+    public String convertToDatabaseColumn(PaymentPolicy policy) {
```

Comment:
wow - a czemu po prostu nie mieć w Driverze enuma, który mówi jaki jest jego typ a w serwisach aplikować odpowiednią policy?

Reply:
mi tam to, jak to zostało zrobione, się podoba ;)